<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopGeneratorDialog</class>
 <widget class="QDialog" name="PopGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="raceLbl">
     <property name="text">
      <string>Race</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="raceIn"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="religionLbl">
     <property name="text">
      <string>Religion</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="religionIn"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="areaLbl">
     <property name="text">
      <string>Area</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="areaIn"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="quantityLbl">
     <property name="text">
      <string>Quantity</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="quantityIn">
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="addPops">
     <property name="text">
      <string>Create pops</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
